utah-design-system-website/src/react/components/websiteContent/library/components/containers/TabGroup/TabGroupPropsDocumentation.jsx

```diff
@@ -0,0 +1,277 @@
+  return (
+    <>
+      <h3><code>TabGroup</code></h3>
+      <div className="documentation-content--small-text static-example static-example--blank">
+        <TableWrapper className="static-example__component-wrapper">
+          <Table className="table--lines-x">
+            <TableHead>
+              <TableHeadRow>
+                <TableHeadCell className="text-left">Property</TableHeadCell>
+                <TableHeadCell className="text-left">Type</TableHeadCell>
+                <TableHeadCell className="text-left">Default</TableHeadCell>
+                <TableHeadCell className="text-left">Description</TableHeadCell>
+              </TableHeadRow>
+            </TableHead>
+            <TableBody>
+              <TableRow>
+                <TableCell><code className="primary-color">children</code></TableCell>
+                <TableCell>
+                  <div className="props-code-wrapper">
+                    <code>node</code>
+                  </div>
+                </TableCell>
+                <TableCell><em>required</em></TableCell>
+                <TableCell>
+                  The content of the tab group.<br />
+                  Includes <code>TabGroupTitle</code>, <code>TabList</code>, <code>Tab</code>, <code>TabPanels</code> and <code>TabPanel</code>.
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell><code className="primary-color">className</code></TableCell>
+                <TableCell><code>string</code></TableCell>
+                <TableCell>null</TableCell>
+                <TableCell>
+                  The CSS Class to use on the tab group wrapper.
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell><code className="primary-color">defaultValue</code></TableCell>
+                <TableCell><code>number</code></TableCell>
+                <TableCell>null</TableCell>
+                <TableCell>
+                  Which tab should be selected by default.<br />
+                  Must match the corresponding tab <code>id</code>.
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell><code className="primary-color">isVertical</code></TableCell>
+                <TableCell><code>boolean</code></TableCell>
+                <TableCell><code>false</code></TableCell>
+                <TableCell>
+                  Change the orientation of the tab group.
+                </TableCell>
+              </TableRow>
+              <TableRow>
+                <TableCell><code className="primary-color">onChange</code></TableCell>
+                <TableCell>
+                  <div className="props-code-wrapper">
+                    <code>function</code>
+                  </div>
+                </TableCell>
+                <TableCell>null</TableCell>
+                <TableCell>The function to call when a tab is clicked.</TableCell>
```

Comment:
Maybe add that the function is passed the new selected tab's tabId as the value?